spock-specs/src/test/groovy/org/spockframework/runtime/GlobalExtensionRegistrySpec.groovy

```diff
@@ -123,29 +136,37 @@ class GlobalExtensionRegistrySpec extends Specification {
     registry.globalExtensions[0].settings.is(registry.globalExtensions[1].settings)
   }
 
-  def "allows to configure local extensions"() {
+  def "allows to instantiate local extensions via #desc injection"(Class clazz) {
```

Comment:
Any reason you declare the parameters explicitly?
We usually don't do this, do we?

Reply:
Habit from work